src/nrnoc/treeset.cpp

```diff
@@ -1603,10 +1602,6 @@ static double diam_from_list(Section* sec, int inode, Prop* p, double rparent)
 
 void v_setup_vectors(void) {
     int inode, i;
-    int isec;
-    Section* sec;
-    Node* nd;
-    Prop* p;
```

Comment:
very hard to review. huge scope with multiple `#if` directives. couldn't spot anything, but i typically wouldn't.

Reply:
I checked again